public/homepage/scripts/main.js

```diff
@@ -17,6 +17,7 @@ require.config({
     "uuid": "/node_modules/node-uuid/uuid",
     "cookies": "/node_modules/cookies-js/dist/cookies",
     "analytics": "/node_modules/webmaker-analytics/analytics",
+    "gallery": "/{{ locale }}/homepage/scripts/gallery",
```

Comment:
If you're creating it as a module here, you'll also need to add it to the Gruntfile here: https://github.com/mozilla/thimble.mozilla.org/blob/master/Gruntfile.js#L37-L50
`gallery: "homepage/scripts/gallery"`